### DISCUSSION TITLE
ChatGPT Docs integration

### DISCUSSION THREAD

**Discussion by fellz** [nuxt/nuxt#24312] [General]
Hello
Since chatGPT opened new tool GPT Builder and there is knowledge option here can you add nuxt.pdf or ( docs.pdf) to the project for documentation reference. There are some tools to parse docs on the site and docs folder on the github repo but i think it's simplest and fastest solution for now.

p.s. i found similar issue here https://github.com/nuxt/nuxt/issues/22715 but i don't see any implementation of this idea so far.

**Comment by rubynouille** [comment]
I made a GPT with the documentation of Nuxt along with a couple modules [here](https://chat.openai.com/g/g-QpUGlIzio-nuxt-code-architect) (Code Architect for Nuxt) for this, and I'd love to discuss it with the team! Updating the documentation as often as possible is a struggle, especially with the bunch of modules it's covering. Would simplify the process by a lot.
